src/vision/scripts/pyimagesearch/panorama.py

```diff
@@ -0,0 +1,103 @@
+# import the necessary packages
+import numpy as np
+import imutils
+import cv2
+
+class Stitcher:
+	def __init__(self):
+		# determine if we are using OpenCV v3.X and initialize the
+		# cached homography matrix
+		self.isv3 = imutils.is_cv3()
+		self.cachedH = None
+
+	def stitch(self, images, ratio=0.75, reprojThresh=4.0):
+		# unpack the images
+		(imageB, imageA) = images
+
+		# if the cached homography matrix is None, then we need to
+		# apply keypoint matching to construct it
+		if self.cachedH is None:
+			# detect keypoints and extract
+			(kpsA, featuresA) = self.detectAndDescribe(imageA)
+			(kpsB, featuresB) = self.detectAndDescribe(imageB)
+
+			# match features between the two images
+			M = self.matchKeypoints(kpsA, kpsB,
+				featuresA, featuresB, ratio, reprojThresh)
+
+			# if the match is None, then there aren't enough matched
+			# keypoints to create a panorama
+			if M is None:
+				print("Cannot stitched")
```

Comment:
Again, useful for debugging, but we should remove this before we merge to avoid cluttering the output